src/css/archwizard.scss

```diff
@@ -1,9 +1,97 @@
-// color definitions
-$wz-color-default: #E6E6E6 !default;
-$wz-color-current: #808080 !default;
-$wz-color-done: #339933 !default;
-$wz-color-optional: #38ef38 !default;
-$wz-color-editing: #FF0000 !default;
+// Wizard layouts to be generated (all by default).
+// Override this variable to exclude layouts you don't need.
+$wz-layouts: 'horizontal' 'vertical' !default;
+
+// Wizard indicator styles to be generated (all by default).
+// Override this variable to exclude indicator styles you don't use.
+$wz-indicator-styles: 'small', 'large-filled', 'large-empty', 'large-filled-symbols', 'large-empty-symbols' !default;
+
+// Color definitions
+//
+// This is a mapping from indicator style and wizard step state to colors.
+//
+// Predefined indicator styles are: 'small', 'large-filled', 'large-empty', 'large-filled-symbols', 'large-empty-symbols'.
+//
+// Wizard step states are: 'default', 'current', 'done', 'optional', 'editing'.
+//
+// Colors are: 'border-color-default', 'background-color-default', 'symbol-color-default', 'border-color-hover',
+// 'background-color-hover', 'symbol-color-hover'.
+//
+// Notes:
+// - The order of wizard step states in this mapping does not affect the order of resulting CSS rules, which is fixed
+//   internally by angular-archwizard.
+// - Depending on the active indicator style, some colors may not be used.  For example, 'background-color-default' and
+//   'background-color-hover' do not make sense in 'large-empty-symbols' indicator style.
+// - You can set a color to `null` to exclude the corresponding property from CSS output.
+//
+// Default colors
+// --------------
+//
+// You can set default colors by using '_' in place of an indicator style or a wizard step state.
+// The `wz-get-color` function looks up the requested color in the following order, picking the first defined entry:
+//
+// - $wz-colors[$indicator-style][$step-state][$color]
+// - $wz-colors[$indicator-style]['_'][$color]
+// - $wz-colors['_'][$step-state][$color]
+// - $wz-colors['_']['_'][$color]
+//
+$wz-colors: (
+  '_': (
+    'default': (
+      'border-color-default': #E6E6E6,

```

Comment:
I think we need some place where we document what each of these values means/defines. E.g.:

```
'border-color-default': The border color used for a wizard step in the default state (i.e. unselected and not yet completed)
...
```

Reply:
I think it will result in too much comments and make it hard to see the code among them. The `$wz-colors` object is already elaborately documented, so from here

```
$wz-colors: (
  '_': (
    'default': (
      'border-color-default': #E6E6E6,
```

it should be understandable that '_' stands for any indicator style and 'default' is a wizard step state. What 'border-color' means is obvious. I added a paragraph about '-hover' and '-default' colors in 802e20f.

Reply:
Ok